app/electron/main.ts

```diff
@@ -453,6 +453,42 @@ class PluginManagerEventListeners {
   }
 }
 
+function getShellEnv(): NodeJS.ProcessEnv {
+  if (process.platform === 'win32') {
```

Comment:
btw, I think the only reason we should read env vars from a shell is if zsh is being used. For other shells they already have env vars set.

Since zsh can be used on linux and even windows I guess.

Reply:
even for bash the env setup can be present in `bashrc` right? Ill handle the case for windows also

Reply:
Windows with bash or other shells can be handled in another future PR if it's an issue.